src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -4352,6 +4352,60 @@ function Test-ApplicationGatewayFirewallPolicyWithUppercaseTransform
 	}
 }
 
+function Test-ApplicationGatewayFirewallPolicyWithInspectionLimit
+{
+	# Setup
+	$location = Get-ProviderLocation "Microsoft.Network/applicationGateways" "West US 2"
+
+	$rgname = Get-ResourceGroupName
+	$wafPolicy = Get-ResourceName
+
+	try
+	{
+		$resourceGroup = New-AzResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "APPGw tag"}
+
+		# WAF Policy and Custom Rule
+		$variable = New-AzApplicationGatewayFirewallMatchVariable -VariableName RequestHeaders -Selector Content-Length
+		$condition =  New-AzApplicationGatewayFirewallCondition -MatchVariable $variable -Operator GreaterThan -MatchValue 1000 -Transform Uppercase -NegationCondition $False
+		$rule = New-AzApplicationGatewayFirewallCustomRule -Name example -Priority 2 -RuleType MatchRule -MatchCondition $condition -Action Block
+		$policySettings = New-AzApplicationGatewayFirewallPolicySetting -Mode Prevention -State Enabled -RequestBodyEnforcement $True -RequestBodyInspectLimitInKB 2000 -MaxFileUploadInMb 70 -FileUploadEnforcement $True -MaxRequestBodySizeInKb 70
```

Comment:
Can we put non default values "false" for the new boolean fields so that we can make sure that they are set. Default value ("true") will be set any ways and so might not be a true test.

Reply:
I think in the swagger the default value is set to true

Reply:
"requestBodyEnforcement": {
          "type": "boolean",
          "default": true,
          "description": "Whether allow WAF to enforce request body limits."
        },
        "maxRequestBodySizeInKb": {
          "type": "integer",
          "format": "int32",
          "minimum": 8,
          "exclusiveMinimum": false,
          "description": "Maximum request body size in Kb for WAF."
        },
        "fileUploadEnforcement": {
          "type": "boolean",
          "default": true,
          "description": "Whether allow WAF to enforce file upload limits."
        },

Reply:
@Shawnli222 agree with what you are saying. What I am asking is that when are trying to put a test case which will set a value, we should try to set the non-default value since default value will anyways be set.

Reply:
oh I see, will make this change